packages/client/hmi-client/src/components/workflow/ops/tera-node-charts.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+	<tera-progress-spinner v-if="isLoading" :font-size="2" is-centered style="height: 100%">
```

Comment:
If I am understanding correctly this spinner is for the node running not the charts loading data
It is unintuitive to me to have this spinner hidden in this component
I prefer these spinners to be owned by the node. Nodes tend to be quite simple I dont think saving 3 lines per node is worth it here 

I am open to having my mind changed

Reply:
yeah I think the spinner needs to be on the node not inside here. While it's loading, we won't even render the charts.  And without the spinner, I wonder if we need this component at all.